fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/query/FDBRecordStoreQueryTest.java

```diff
@@ -239,7 +241,7 @@ public void queryWithContinuation() throws Exception {
                 if (retrieved.size() > 100) {
                     fail("added more records than were present");
                 }
-                continuation = cursor.getContinuation();
+                continuation = cursor.onNext().join().getContinuation().toBytes();
```

Comment:
Should probably be `getNext()` instead of `onNext().join()`. This might be from my work before we had `getNext()`.

Reply:
Done.

Reply:
I went through the code base looking for instances of `onNext().join()`, and I think I got them in the next commit.